Examples/CMakeLists.txt

```diff
@@ -16,7 +16,3 @@ add_subdirectory(Segmentation)
 add_subdirectory(SpatialObjects)
 add_subdirectory(Statistics)
 add_subdirectory(RegistrationITKv4)
-
-if(ITKV3_COMPATIBILITY)
```

Comment:
I think we concluded that examples for v3 registration should remain. Just remove this `if` condition, and keep the examples.